settings/DevHome.Settings/ViewModels/ExperimentalFeaturesViewModel.cs

```diff
@@ -27,4 +29,15 @@ public ExperimentalFeaturesViewModel(IExperimentationService experimentationServ
             new(stringResource.GetLocalized("Settings_ExperimentalFeatures_Header"), typeof(ExperimentalFeaturesViewModel).FullName!),
         };
     }
+
+    // A crude function to check for feature presence

```

Comment:
Maybe say more here about how features that might not be on the machine need to be added here,